projects/components/src/overlay/sheet/sheet-overlay.component.ts

```diff
@@ -58,7 +63,7 @@ import { SheetOverlayConfig, SheetSize } from './sheet';
   `
 })
 export class SheetOverlayComponent {
-  public readonly sheetTitle: string;
+  public readonly sheetTitle: string | TemplateRef<unknown>;
```

Comment:
Title is supposed to be text only. If we allow it to be a template then technically we can pass any custom content here. I would propose that we keep title as it is and introduce a new property `customHeaderTemplate` that will be rendered between the title text and the header controls